packages/ui/components/pages/Fuse/Modals/Terms.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    Button,
+    UnorderedList,
+    ListItem,
+    List,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Text,
+  } from '@chakra-ui/react'
+
+  import React from 'react';
+  import { useState, useEffect } from 'react';
+
+
+  const Terms = () => {
+
+    const [hasAcceptedTerms, setHasAcceptedTerms] = useState<boolean>(Boolean(document.cookie.split('; ').find((row) => row.startsWith('MidasTandCAccepted='))?.split('=')[1]));
+
+    let isOpen = !hasAcceptedTerms;
+    const accept = () => {
+      document.cookie = 'MidasTandCAccepted=true;';
```

Comment:
Using a `cookie` works here but I think `localStorage` would be the way to go here. Cookies get sent back and forth between front & backend. This is not any of the requirement we need here.

https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage